docs/usage/private-container-reg.md

```diff
@@ -0,0 +1,5 @@
+# Private container registries
+
+To access a private container registry, follow the steps on [this page](http://kubernetes.io/docs/user-guide/images/).
+
+We recommend you use ImagePullSecrets, but if you would like to configure access on the minishift VM you can place the `.dockercfg` in the `/home/docker` directory or the `config.json` in the `/home/docker/.docker` directory.
```

Comment:
@Preeticp not needed right now, we can figure it out later on but it's not critical and the content of this (and other) topics might still change a lot between now and release time.